indico/modules/events/contributions/templates/management/contributions.html

```diff
@@ -93,6 +93,17 @@
                         {%- trans %}Export{% endtrans -%}
                     </a>
                     <ul class="dropdown">
+                        <li>
+                            <a href ="#" class="icon-file-pdf js-enable-if-checked disabled"
+                               data-href="{{ url_for('.contributions_tex_export_dialog', event) }}"
+                               data-method="POST"
+                               data-params-selector="#contribution-list input[name=contribution_id]:checked"
+                               data-ajax-dialog
+                             >
+                                {%- trans %}Export via LaTeX{% endtrans -%}
```

Comment:
"LaTeX package" or something like that. "Export" is redundant since the button is already  labelled "Export"

Reply:
Done